resources/en/vocab/Field.vocab.bxb

```diff
@@ -0,0 +1,5 @@
+vocab (self.Field) {
+  "firstAndLastName" {
```

Comment:
See if we can add documentation about how vocab is layered for Enums.

Reply:
done. thx. https://sixfivelabs.myjetbrains.com/youtrack/issue/DOC-1179